machinepool_test.go

```diff
@@ -167,3 +175,27 @@ func Test_MachinePoolCR(t *testing.T) {
 		}
 	}
 }
+
+func findSpark(ctx context.Context, client ctrl.Client, clusterID, machinePoolID string) (*v1alpha1.Spark, error) {
+	var spark *v1alpha1.Spark
+	{
+		var sparkList v1alpha1.SparkList
+		err := client.List(ctx, &sparkList, ctrl.MatchingLabels{label.Cluster: clusterID})
```

Comment:
Should we rather use the CAPI `ClusterName` label to be future proof?

Reply:
Yeah, good idea.

Reply:
Done.